conf/messages

```diff
@@ -3314,10 +3314,21 @@ yourSelfAssessmentChargeSummary.howIsInterestCalculated.p6          = 250,000 /
 yourSelfAssessmentChargeSummary.howIsInterestCalculated.p7          = 684.93 x 0.0725 = 49.66
 
 yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.linkText     = What is payment on account?
-yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.p1           = ‘Payments on account’ are two advance payments made towards your next tax bill.
-yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.p2           = HMRC estimates the total amount based on your previous year’s tax bill. Each payment is half of that amount.
 yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.p3           = If you make an amendment to the tax return and the tax bill changes, then each payment on account will be updated to half of the new total.
 yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.p4           = Help with understanding your Self Assessment tax bill
+yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.p1           = Payments on account are two advance payments made towards your next tax bill. They pay for:
+yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.p2           = HMRC estimates the total amount based on your previous years tax bill. Each payment is half of that amount.
+yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.bullet1      = Income Tax
+yourSelfAssessmentChargeSummary.whatIsPaymentOnAccount.bullet2      = Class 4 National Insurance contributions (opens in new tab)
+yourSelfAssessmentChargeSummary.selfAssessmentCharges.linkText.part1 = View
+yourSelfAssessmentChargeSummary.selfAssessmentCharges.linkText.part2 = your Self Assessment charges
+yourSelfAssessmentChargeSummary.selfAssessmentCharges.linkText-agent.part1 = View
+yourSelfAssessmentChargeSummary.selfAssessmentCharges.linkText-agent.part2 = your client’s Self Assessment charges
+yourSelfAssessmentChargeSummary.chargeHistory.poa1CodedOut.text.heading = First payment on account history
+yourSelfAssessmentChargeSummary.chargeHistory.poa2CodedOut.text.heading = Second payment on account history
+yourSelfAssessmentChargeSummary.chargeHistory.caption.poa1CodedOut.text.heading = This first payment on account goes towards your {0} to {1} tax bill.
+yourSelfAssessmentChargeSummary.chargeHistory.caption.poa2CodedOut.text.heading = This second payment on account goes towards your {0} to {1} tax bill.
+yourSelfAssessmentChargeSummary.chargeHistory.amend.codingOut.text = Amount adjusted to be collected through your PAYE tax code in {0} to {1} tax year.
```

Comment:
Can you remove the full stop at the end of these two? Other entries in the table don't have them